docs/application/toc_application.md

```diff
@@ -1215,7 +1215,7 @@
 ##### [Device/Resource Model Manager](/application/tizen-studio/rt-ide/smartths-ext/model-manager.md)
 
 ### Release Notes
-
+#### [3.1 (Jan 15, 2019)](/application/tizen-studio/release-notes/3-1-release-notes.md)
```

Comment:
This toc_application.md file is to backup the toc_all.md file version of #512.
You don't need to update toc_application.md file.